nndownload/nndownload.py

```diff
@@ -2005,170 +2005,134 @@ def download_video_comments(
     filename = replace_extension(filename, "comments.json")
 
     if os.path.exists(filename):
-        output(f"File {filename} exists. Skipping...\n", logging.INFO)
+        output(f"Comments file \"{filename}\" exists. Skipping...\n", logging.INFO)
         return False
 
-    last_time: int = _CMDL_OPTS.comments_from
-    # sys.maxsize is practically big enough (no videos on niconico have reached 100 mil comments)
-    comments_limit: int = sys.maxsize if _CMDL_OPTS.request_all_comments else _CMDL_OPTS.comments_limit
```

Comment:
It's never a good idea to rely on kludges like a max int. Instead, overwrite the limit with `None` if the user requests all comments.